samples/apps/autogen-studio/autogenstudio/workflowmanager.py

```diff
@@ -620,7 +712,7 @@ def _run_workflow(self, message: str, history: Optional[List[Message]] = None, c
         user_proxy = {
             "config": {
                 "name": "user_proxy",
-                "human_input_mode": "NEVER",
```

Comment:
BTW, this was just an experiment.  I forgot to reverse it before submitting the PR.  Final solution doesn't need this.